.github/SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Supported Versions
+
+The following versions of Wirespec are currently being supported with security updates:
+
+| Version | Supported          |
+|---------|--------------------|
+| Latest  | :white_check_mark: |
+
+We recommend always using the latest version of Wirespec to ensure you have all security patches and updates.
+
+## Reporting a Vulnerability
+
+1. **DO NOT** create a public GitHub issue for the vulnerability.
+2. Email your findings to [info@flock.community](mailto:info@flock.community).
```

Comment:
Can we use wirespec@flock.community for this instead? I can help you create that e-mail.